votor/src/voting_loop.rs

```diff
@@ -698,15 +696,15 @@ impl VotingLoop {
         if voting_context.vote_history.its_over(slot) {
             return false;
         }
-        for (block_id, bank_hash) in blocks.into_iter() {
+        for block_id in blocks.into_iter() {
             if voting_context
                 .vote_history
-                .voted_notar_fallback(slot, block_id, bank_hash)
+                .voted_notar_fallback(slot, block_id)
             {
                 continue;
             }
-            info!("{my_pubkey}: Voting notarize fallback for slot {slot} hash {bank_hash} block_id {block_id}");
-            let vote = Vote::new_notarization_fallback_vote(slot, block_id, bank_hash);
+            info!("{my_pubkey}: Voting notarize fallback for slot {slot} block_id {block_id}");
+            let vote = Vote::new_notarization_fallback_vote(slot, block_id, Hash::default());
```

Comment:
@AshwinSekar is this safe to remove from votes or do we still need bank hashes to verify votes in the vote program

Reply:
I think you need to remove from vote program as well? When we move the vote definitions to the new place we will probably just copy the whole file over.

Reply:
yup, addressed in the PR description, I think the vote program can be updated separately